uiManager.js

```diff
@@ -0,0 +1,43 @@
+const backToTopButton = document.querySelector(".back-to-top-arrow");
```

Comment:
This file name does not seem to follow naming conventions for the web, probably it should be renamed to `ui_manager.js`

Reply:
My apologies. I'll fix that.